Core.Collectors/Config/StorageManager.cs

```diff
@@ -99,15 +99,13 @@ public StorageManager(JArray recordWritersArray, ITelemetryClient telemetryClien
             }
 
             JToken storageConnectionEnvironmentVariableToken = recordWriterToken.SelectToken("StorageConnectionEnvironmentVariable");
-            JToken notificationQueueEnvironmentVariableToken = recordWriterToken.SelectToken("NotificationQueueEnvironmentVariable");
 
             // The following are optional (only used in Azure DevOps right now), so permit these values (tokens) to be null.
             string storageConnectionEnvironmentVariable = storageConnectionEnvironmentVariableToken?.Value<string>();
-            string notificationQueueEnvironmentVariable = notificationQueueEnvironmentVariableToken?.Value<string>();
 
             string rootContainer = rootContainerToken.Value<string>();
             string outputQueueName = outputQueueNameToken.Value<string>();
-            return this.ConstructAzureBlobWriter(rootContainer, outputQueueName, identifier, this.telemetryClient, functionContext, contextWriter, storageConnectionEnvironmentVariable, notificationQueueEnvironmentVariable);
```

Comment:
>storageConnectionEnvironmentVariable, notificationQueueEnvironmentVariable [](http://example.com/codeflow?start=147&length=74)

Why are we removing this?

Reply:
Restored this piece of code for notification queues as connection string arise from router settings. Modifying this to use managed identities would involve infrastructure change as I understand this, so reverted to original.